events/javaone/2015/index.html.slim

```diff
@@ -75,8 +75,8 @@ title: JavaOne
                 .row.talk-info
                   .columns class="#{speaker.url.nil? ? 'large-24' : 'large-13'}"
                     h4 #{speaker.title}
-                  - unless !speaker.url  
-                    .large-11.columns(href="#{speaker.url}")
+                  - unless !speaker.url
+                    .large-11.columns(href="#{site.base_url}#{speaker.url}")
```

Comment:
It's safer to put a '/' between so that it doesn't break if 'speaker.url' is missing the preceding '/'. Change to:

      "#{site.base_url}/#{speaker.url}"